packages/neutrino/bin/start.js

```diff
@@ -53,10 +54,10 @@ module.exports = (middleware, args) => {
         compiler.plugin('done', () => {
           building.succeed('Build completed');
         });
-        compiler.plugin('compile', () => {
+        compiler.plugin('compile', debounce(() => {
```

Comment:
Could you add a comment explaining why this is needed? (I thought Ora did debouncing? https://github.com/sindresorhus/ora#interval)

Reply:
This isn't debouncing the animation, rather this is debouncing the `compile` event. If `devServer` gets several updates in a row, like an editor writes multiple files at different times, the compiler will send several `compile` events, which causes a lot of noise in the terminal. This tries to reduce that logging noise to just once per second.

Reply:
I'll add comment. :)